POC/ColorPicker/ColorPicker/TransparentWindow.xaml

```diff
@@ -9,6 +9,20 @@
         WindowStyle="None"
         ShowInTaskbar="False"
         Background="#01000000"
+        Loaded="OnLoad"
         MouseDown="OnMouseDown"
-        KeyDown="OnKeyDown">
+        KeyDown="OnKeyDown"
+        MouseMove="OnMouseMove">
+    <Window.Resources>
+        <SolidColorBrush x:Key="MagnifiedImageBorderColor" Color="#FF808080"/>
+        <SolidColorBrush x:Key="MagnifiedImagePixelBorderColor" Color="#FFFF0000"/>
+    </Window.Resources>
+    <Canvas>
+        <Border Name="MouseFollower" BorderBrush="{StaticResource MagnifiedImageBorderColor}" BorderThickness="2" CornerRadius="2" Visibility="Hidden">
+            <Grid Height="99" Width="99">
+                <Image Name="MagnifiedImage" RenderOptions.BitmapScalingMode="NearestNeighbor"/>
+                <Rectangle Width="11" Height="11" Stroke="{StaticResource MagnifiedImagePixelBorderColor}" StrokeThickness="1.5"/>
```

Comment:
Is the width/height here related to the Magniification_width/height in the cs file?

Reply:
No, it's simply the dimensions of the pixel "crosshair", which could reasonably be bigger or smaller depending on preference.